spacepy/toolbox/__init__.py

```diff
@@ -2209,18 +2242,23 @@ def query_yes_no(question, default="yes"):
     The "answer" return value is one of "yes" or "no".
 
     Parameters
-    ==========
+    ----------
     question : string
         the question to ask
     default : string (optional)
 
+    Raises
+    ------
+    ValueError
+        The default answer is invalid.
+
```

Comment:
As above, I think it's better to document the valid defaults!

Reply:
This whole function is a mess.... updated to have what is allowed.

Reply:
There's an explicit format for listing available options; if this is used in the parameters then the Raises can go away:
`default : { 'yes', 'no', None} (optional)`

Also, this isn't your change but as long as you're fixing things, the type is `str` not `string`.

Reply:
updated

Reply:
good in 1aa9067e02c6d8d26ecd3023fbc349c3cd062df2